test/functional/tv/tvos-e2e-specs.js

```diff
@@ -28,7 +28,7 @@ describe('tvOS', function () {
     baseCaps = Object.assign({}, TVOS_CAPS, {udid});
     caps = Object.assign({usePrebuiltWDA: true}, baseCaps);
   });
-  after(async function () {
+  afterEach(async function () {
```

Comment:
Please review what should be done before and after each test case. Simulator shutdown and recreation is an expensive operation, which may significantly hit tests performance

Reply:
yeah